### PR TITLE
Document loanbook aggregation

### PR DESCRIPTION
**Pull request by jacobvjk**
- add documentation of exposure-weighted aggregate alignment metric on the loan book level
- publish documentation with pkgdown
- add some interpretation for the company level metrics

### CODE REVIEW

**Comment by jacobvjk** [comment]
@jdhoffa I followed the `pkgdown` documentation [here](https://pkgdown.r-lib.org/articles/pkgdown.html) for a basic attempt at publishing the vignettes on a GH page. I works locally, but on the feature branch, I do not see the website. Not sure if this is expected and only works once it is merged into `main`, but maybe you know more about it?

File: .github/workflows/pkgdown.yaml
Comment (jdhoffa) [inline .github/workflows/pkgdown.yaml:7]: @jacobvjk see here, the website is only deployed when merged to master or main

File: vignettes/company_alignment_metric.Rmd
Comment (jdhoffa) [inline vignettes/company_alignment_metric.Rmd:122]: Typo: `understnad`

**Review by jdhoffa — APPROVED** [review]
I think this LGTM. 

I would suggest merging and, once it is on gh pages and in a more human readable format, having Nick review.

File: vignettes/company_alignment_metric.Rmd
Comment (jacobvjk) [reply vignettes/company_alignment_metric.Rmd]: thanks, fixed

**Comment by jacobvjk** [comment]
> I think this LGTM.
> 
> I would suggest merging and, once it is on gh pages and in a more human readable format, having Nick review.

absolutely, all recent documentation still needs a round of review/approval by Nick

**Comment by jdhoffa** [comment]
Ah can I ask something after the fact? 

Can you use this as the pkgdown template instead of flatly?
https://github.com/RMI-PACTA/pacta.pkgdown.template

**Pull request merged**
